pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1365,6 +1367,17 @@ func (l *LibvirtDomainManager) GetGuestInfo() (v1.VirtualMachineInstanceGuestAge
 	return guestInfo, nil
 }
 
+// InterfacesStatus returns the interfaces Guest Agent reported
+func (l *LibvirtDomainManager) InterfacesStatus(domainInterfaces []api.Interface) []api.InterfaceStatus {
+	interfaces := l.agentData.GetInterfaceStatus()
+	return agentpoller.MergeAgentStatusesWithDomainData(domainInterfaces, interfaces)
```

Comment:
In the [client](https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-launcher/notify-client/client.go#L388) we have a check `domainInterfaces != nil`. don't we need it here as well?

Reply:
It can't be nil here, 
because how https://github.com/kubevirt/kubevirt/pull/5428/files#diff-510b183b2d5d079863708a79f37a82f552d485d8c3e6e10e5e2eefecfc946edaR134 is implemented

btw even on the client that you mentioned its not that easy becomes nil, but this is other story

Reply:
Testing the first time, before the first assignment, in order to make sure